apps/nextjs-website/src/components/atoms/CtaSlide/CtaSlide.tsx

```diff
@@ -67,6 +67,7 @@ const CtaSlide = ({
           src={backgroundImage}
           alt={title}
           fill={true}
+          priority
```

Comment:
Why did you add the priority prop to the Hero Image?

Reply:
I addded it because there was a warning relative to it, and the `priority` resolves the issue, but we can remove it if we don't need it.
<img width="834" alt="Screenshot 2024-01-09 alle 14 46 19" src="https://github.com/pagopa/developer-portal/assets/146933303/67569b24-498d-4095-ba05-77dd196f0303">

Reply:
Please remove it from this PR, add a task to the board, and open a new PR